tests/network/vmi_multus.go

```diff
@@ -679,6 +679,7 @@ var _ = Describe("[Serial]SRIOV", func() {
 		getSriovVmi := func(networks []string, cloudInitNetworkData string) *v1.VirtualMachineInstance {
 
 			withVmiOptions := []libvmi.Option{
+				libvmi.WithCloudInitNoCloudUserData(tests.GetFedoraToolsGuestAgentUserData(), false),
```

Comment:
if you keep the sed on the kubevirtci, we need to remove please the setting of the password from `GetFedoraToolsGuestAgentUserData`
as its redundant and also prone for races i think (password exists and recreated)